css/src/atomics/width.scss

```diff
@@ -0,0 +1,37 @@
+@use 'sass:list';
+
+$universal-widths: auto 100, 150, 200, 250, 300, 350 !default;
```

Comment:
Comma is missed after `auto`

Reply:
Also, probably it shouldn't be here. Per rules below we end up with `width: autopx !important`. Add .`width-auto` directly after every `@each` rule set.